frontend/src/assets/config/dev.json

```diff
@@ -1,6 +1,28 @@
 {
     "api": {
-        "apiBase": "https://svmeal-api.jmnw.me/",
-        "context": "api"
+        "defaultApiName": "outcobra",
+        "apis": [
+            {
+                "name": "outcobra",
+                "apiBase": "http://localhost/",
+                "context": "api",
```

Comment:
I think we should just put this in `apiBase`. Is there a reason to put it in a separate variable?